src/Orchard.Web/Core/Settings/Drivers/SiteSettingsPartDriver.cs

```diff
@@ -69,11 +69,12 @@ public class SiteSettingsPartDriver : ContentPartDriver<SiteSettingsPart> {
 
             var previousBaseUrl = model.Site.BaseUrl;
 
-            updater.TryUpdateModel(model, Prefix, null, new [] { "Site.SuperUser", "Site.MaxPageSize" });

```

Comment:
This and the other SiteSettingsPart-related modifications don't seem to have anything to do with the issue.